src/app/components/common/modal/modal.component.scss

```diff
@@ -112,3 +112,10 @@ body.jw-modal-open .slide-in-form .jw-modal-body {
   /* body overflow is hidden to hide main scrollbar when modal window is open */
   overflow: hidden;
 }
+
+@media screen and (max-width: 768px) {
```

Comment:
We now have both media queries and breakpoint values as variables, see `cssvars.scss`.
The breakpoints there [are based on material numbers.](https://material-ui.com/ru/customization/breakpoints/)